cmd/kube-apiserver/app/OWNERS

```diff
@@ -0,0 +1,41 @@
+reviewers:
```

Comment:
Or actually, we can just remove this file if we want to use the owners file from the parent directory